src/tests/harness/BenchmarkDotNet.Extensions.Tests/UniqueValuesGeneratorTests.cs

```diff
@@ -72,5 +72,115 @@ public void GetNonDefaultValueReturnsNonDefaultValue()
             Assert.NotEqual(default(int), ValuesGenerator.GetNonDefaultValue<int>());
             Assert.NotEqual(default(string), ValuesGenerator.GetNonDefaultValue<string>());
         }
+
+        [Fact]
+        public void SupportsByte()
+        {
+            Supports<byte>();
+        }
```

Comment:
nit: you could use expressions for all the one-liners:

```suggestion
        public void SupportsByte() => Supports<byte>();
```

Reply:
Good idea!